### ISSUE TITLE
Error :: Call to undefined function et_fb_enabled() :: v3.10+

### ISSUE THREAD

**Issue by albiurs** [elegantthemes/Divi-Beta#1117]
### Problem:
I get the same kind of errors as reported here (https://github.com/elegantthemes/Divi-Beta/issues/1102) by praveen-tast. However, as the error refers to another line in framework.php then the original issue reported by lots0logs, I decided to post a new one...

I get the following php error logged for several month on different web servers:
`
[28-Oct-2018 21:43:05 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function et_fb_enabled() in /home/[user]/public_html/[domain]/htdocs/wp-content/themes/Divi/includes/builder/framework.php:224
Stack trace:
#0 /home/[user]/public_html/[domain]/htdocs/wp-includes/class-wp-hook.php(286): et_builder_get_modules_js_data('')
#1 /home/[user]/public_html/[domain]/htdocs/wp-includes/class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 /home/[user]/public_html/[domain]/htdocs/wp-includes/plugin.php(453): WP_Hook->do_action(Array)
#3 /home/[user]/public_html/[domain]/htdocs/wp-includes/general-template.php(2630): do_action('wp_footer')
#4 /home/[user]/public_html/[domain]/htdocs/wp-content/themes/Divi/footer.php(58): wp_footer()
#5 /home/[user]/public_html/[domain]/htdocs/wp-includes/template.php(688): require_once('/home/[user]/...')
#6 /home/[user]/public_html/[domain]/htdocs/wp-includes/template.php(647): load_template('/home/[user]/...', true)
#7 /home/[user]/public_ht in /home/[user]/public_html/[domain]/htdocs/wp-content/themes/Divi/includes/builder/framework.php on line 224
`

### Steps To Reproduce:
The php error gets logged randomly every couple of days (somewhere from 1 day up to 2 weeks in between two logged errors) on productive server. Therefore, I can't deactivate all plugins for such a long time to check for incompatibilities with plugins.

### Additional Information:
I checked all my past logs for the origin, when this error series actually started:
-> The error was logged the very first time at July 17th 2018, after I updated Divi from version 3.9  to 3.10.
-> As I found in the Divi Change Log, the related file (.../Divi/includes/builder/framework.php) was changed in this update. Therefore, I think this update actually introduced the error. 
-> However, the related line in framework.php has changed in one of the recent updates again, so it was .../framework.php:220 until a couple of weeks ago, when the line number changed to .../framework.php:224 due to recent Divi updates after v. 3.16.
A Google search shows that tons of Divi sites actually experience this issue:
https://www.google.com/search?source=hp&ei=Wd7WW5_GBoLKrgTgjpX4DA&q=%22%2Fwp-content%2Fthemes%2FDivi%2Fincludes%2Fbuilder%2Fframework.php+on+line+220%22&oq=%22%2Fwp-content%2Fthemes%2FDivi%2Fincludes%2Fbuilder%2Fframework.php+on+line+220%22&gs_l=psy-ab.3...2611.2611.0.3170.2.1.0.0.0.0.69.69.1.1.0....0...1c.2.64.psy-ab..1.0.0.0...0.17B8b1xBONA

Apart of the error logged, I did not experience any issues with Divi or any plugin. All the sites are running smoothly.

**Web Server**:
Issue on both, Apache and LiteSpeed web servers.

**WordPress**:
Productive WordPress Multisite install.

**Dropped Support Ticket**:
My support ticket was dropped by the support staff due to the lack of reproducibility of the issue. However, according to the big amount of sites showing the issue, I think it's worth to further investigate.

**Comment by etstaging** [comment]
### QA Checklist
<strong>Reporter</strong>
- [x] Helpful title used
- [x] Detailed description
- [ ] Steps to reproduce included <img src="https://assets-cdn.github.com/images/icons/emoji/unicode/274c.png" width="16" height="16"/>
- [ ] Support Ticket URL included
- [ ] Image and/or gif

<strong>Developer</strong>
- [ ] Steps to reproduce proven solved
- [ ] Test included and coverage acceptable and all Tests passing?
- [ ] PR description has layman's description of problem and solution and "Areas Affected" listed/explained?
- [ ] PR branch includes latest commits from master branch (at the time of checking)?
- [ ] Estimated points on issue
- [ ] Proper code documentation and conventions?
- [ ] Proper RTL appearance
- [ ] i18n used if applicable
- [ ] Ajax/Form used? Nonce used? Perms checked? All user input sanitized and validated? Intentional unsanitized/unescaped functions used?
- [ ] Database saved/updated? Data sanitized and validated?
- [ ] Variable output properly escaped and validated?

<strong>Peer Review</strong>
- [ ] PR branch includes latest commits from master branch (at the time of checking)?
- [ ] Steps to reproduce proven solved
- [ ] Affected areas are tested and ok
- [ ] Proper code documentation and conventions
- [ ] Proper RTL appearance
- [ ] i18n used if applicable
- [ ] Test(s) included and coverage acceptable
- [ ] Ajax/Form used? Nonce used? Perms checked? All user input sanitized and validated? Intentional unsanitized/unescaped functions used?
- [ ] Database saved/updated? Data sanitized and validated?
- [ ] Variable output properly escaped and validated?

<strong>Quality Assurance</strong>
- [ ] All tests passed on CircleCI.
- [ ] Steps to reproduce passed, the fix works.
- [ ] No errors in console or output 
- [ ] Affected areas are tested and ok
- [ ] The fix does not change a customer's website in any surprising or unintended way
  - [ ] If Divi's design <em>is</em> adjusted, settings are migrated gracefully and invisibly

<strong>Design Review</strong>
- [ ] UI is to spec
- [ ] UX is to spec
- [ ] Design is to spec
- [ ] No surprising visual changes to a customer's website will be experienced after updating
  - [ ] Any/all visual changes are migrated gracefully

<strong>Dev Review</strong>
- [ ] Test(s) included and coverage acceptable
- [ ] Ajax/Form used? Nonce used? Perms checked? All user input sanitized and validated? Intentional unsanitized/unescaped functions used?
- [ ] Database saved/updated? Data sanitized and validated?
- [ ] Variable output properly escaped and validated? 
- [ ] i18n used if applicable
- [ ] Implementation acceptable, DRY, Scaleable, future safe
- [ ] Best practices used, cleverness and grokage, ES6 use, etc.

**Comment by MIBul** [comment]
Server Details:
Windows Server 2016 Standard; IIS 10; PHP 7.2.1
Wordpress: 4.9.8
Plugins: 
Activity Log Version 2.5.1
Autoptimize - Version 2.4.1 
Bloom - Version 1.3.6
Classic Editor - Version 0.5
Comet Cache - Version 170220
Divi Blog Extras - Version 2.1.3
Duplicate Post - Version 3.2.2
Gravity Forms - Version 2.3.6
Gravity Forms + Custom Post Types - Version 3.1.13
Gravity Forms Multilingual - Version 1.3.17
IP Geo Block - Version 3.0.16
Media Library Folders for WordPress - Version 4.3.0
Monarch Plugin - Version 1.4.8
Popup Maker - Version 1.7.30
Sucuri Security - Auditing, Malware Scanner and Hardening - Version 1.8.18
TablePress - Version 1.9.1
Wordfence Security - Version 7.1.17
WP Security Audit Log - Version 3.2.4
WPML Multilingual CMS - Version 4.0.8
WPML String Translation - Version 2.8.8
WPML Translation Management - Version 2.6.8
Yoast SEO - Version 9.1

Step To Reproduce the Issue:
- within the site -> Enable Visual Builder - > edit a page -> save page; issue does not occur every time

PHP log:
[08-Nov-2018 02:09:51 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function et_fb_enabled() in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php:226
Stack trace:
#0 E:\[domain]\www\wp-includes\class-wp-hook.php(286): et_builder_get_modules_js_data('')
#1 E:\[domain]\www\wp-includes\class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 E:\[domain]\www\wp-includes\plugin.php(453): WP_Hook->do_action(Array)
#3 E:\[domain]\www\wp-includes\general-template.php(2630): do_action('wp_footer')
#4 E:\[domain]\www\wp-content\themes\Divi\footer.php(58): wp_footer()
#5 E:\[domain]\www\wp-includes\template.php(688): require_once('E:\\[domain]\\w...')
#6 E:\[domain]\www\wp-includes\template.php(647): load_template('E:\\[domain]\\w...', true)
#7 E:\[domain]\www\wp-includes\general-template.php(76): locate_template(Array, true)
#8 E:\[domain]\www\wp-content\themes\Divi\404.php(19): get_footer()
#9 E:\[domain]\www\wp-content\plugins\wordfence\lib\wordfenceClass.php(6433): include('E:\\[domain]\\w...')
# in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php on line 226
[08-Nov-2018 02:09:53 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function et_fb_enabled() in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php:226
Stack trace:
#0 E:\[domain]\www\wp-includes\class-wp-hook.php(286): et_builder_get_modules_js_data('')
#1 E:\[domain]\www\wp-includes\class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 E:\[domain]\www\wp-includes\plugin.php(453): WP_Hook->do_action(Array)
#3 E:\[domain]\www\wp-includes\general-template.php(2630): do_action('wp_footer')
#4 E:\[domain]\www\wp-content\themes\Divi\footer.php(58): wp_footer()
#5 E:\[domain]\www\wp-includes\template.php(688): require_once('E:\\[domain]\\w...')
#6 E:\[domain]\www\wp-includes\template.php(647): load_template('E:\\[domain]\\w...', true)
#7 E:\[domain]\www\wp-includes\general-template.php(76): locate_template(Array, true)
#8 E:\[domain]\www\wp-content\themes\Divi\404.php(19): get_footer()
#9 E:\[domain]\www\wp-content\plugins\wordfence\lib\wordfenceClass.php(6433): include('E:\\[domain]\\w...')
# in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php on line 226
[08-Nov-2018 02:09:55 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function et_fb_enabled() in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php:226
Stack trace:
#0 E:\[domain]\www\wp-includes\class-wp-hook.php(286): et_builder_get_modules_js_data('')
#1 E:\[domain]\www\wp-includes\class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 E:\[domain]\www\wp-includes\plugin.php(453): WP_Hook->do_action(Array)
#3 E:\[domain]\www\wp-includes\general-template.php(2630): do_action('wp_footer')
#4 E:\[domain]\www\wp-content\themes\Divi\footer.php(58): wp_footer()
#5 E:\[domain]\www\wp-includes\template.php(688): require_once('E:\\[domain]\\w...')
#6 E:\[domain]\www\wp-includes\template.php(647): load_template('E:\\[domain]\\w...', true)
#7 E:\[domain]\www\wp-includes\general-template.php(76): locate_template(Array, true)
#8 E:\[domain]\www\wp-content\themes\Divi\404.php(19): get_footer()
#9 E:\[domain]\www\wp-content\plugins\wordfence\lib\wordfenceClass.php(6433): include('E:\\[domain]\\w...')
# in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php on line 226
[08-Nov-2018 02:09:57 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function et_fb_enabled() in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php:226
Stack trace:
#0 E:\[domain]\www\wp-includes\class-wp-hook.php(286): et_builder_get_modules_js_data('')
#1 E:\[domain]\www\wp-includes\class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 E:\[domain]\www\wp-includes\plugin.php(453): WP_Hook->do_action(Array)
#3 E:\[domain]\www\wp-includes\general-template.php(2630): do_action('wp_footer')
#4 E:\[domain]\www\wp-content\themes\Divi\footer.php(58): wp_footer()
#5 E:\[domain]\www\wp-includes\template.php(688): require_once('E:\\[domain]\\w...')
#6 E:\[domain]\www\wp-includes\template.php(647): load_template('E:\\[domain]\\w...', true)
#7 E:\[domain]\www\wp-includes\general-template.php(76): locate_template(Array, true)
#8 E:\[domain]\www\wp-content\themes\Divi\404.php(19): get_footer()
#9 E:\[domain]\www\wp-content\plugins\wordfence\lib\wordfenceClass.php(6433): include('E:\\[domain]\\w...')
# in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php on line 226
[08-Nov-2018 02:09:59 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function et_fb_enabled() in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php:226
Stack trace:
#0 E:\[domain]\www\wp-includes\class-wp-hook.php(286): et_builder_get_modules_js_data('')
#1 E:\[domain]\www\wp-includes\class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 E:\[domain]\www\wp-includes\plugin.php(453): WP_Hook->do_action(Array)
#3 E:\[domain]\www\wp-includes\general-template.php(2630): do_action('wp_footer')
#4 E:\[domain]\www\wp-content\themes\Divi\footer.php(58): wp_footer()
#5 E:\[domain]\www\wp-includes\template.php(688): require_once('E:\\[domain]\\w...')
#6 E:\[domain]\www\wp-includes\template.php(647): load_template('E:\\[domain]\\w...', true)
#7 E:\[domain]\www\wp-includes\general-template.php(76): locate_template(Array, true)
#8 E:\[domain]\www\wp-content\themes\Divi\404.php(19): get_footer()
#9 E:\[domain]\www\wp-content\plugins\wordfence\lib\wordfenceClass.php(6433): include('E:\\[domain]\\w...')
# in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php on line 226
[08-Nov-2018 02:10:02 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function et_fb_enabled() in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php:226
Stack trace:
#0 E:\[domain]\www\wp-includes\class-wp-hook.php(286): et_builder_get_modules_js_data('')
#1 E:\[domain]\www\wp-includes\class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 E:\[domain]\www\wp-includes\plugin.php(453): WP_Hook->do_action(Array)
#3 E:\[domain]\www\wp-includes\general-template.php(2630): do_action('wp_footer')
#4 E:\[domain]\www\wp-content\themes\Divi\footer.php(58): wp_footer()
#5 E:\[domain]\www\wp-includes\template.php(688): require_once('E:\\[domain]\\w...')
#6 E:\[domain]\www\wp-includes\template.php(647): load_template('E:\\[domain]\\w...', true)
#7 E:\[domain]\www\wp-includes\general-template.php(76): locate_template(Array, true)
#8 E:\[domain]\www\wp-content\themes\Divi\404.php(19): get_footer()
#9 E:\[domain]\www\wp-content\plugins\wordfence\lib\wordfenceClass.php(6433): include('E:\\[domain]\\w...')
# in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php on line 226
[08-Nov-2018 02:10:06 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function et_fb_enabled() in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php:226
Stack trace:
#0 E:\[domain]\www\wp-includes\class-wp-hook.php(286): et_builder_get_modules_js_data('')
#1 E:\[domain]\www\wp-includes\class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 E:\[domain]\www\wp-includes\plugin.php(453): WP_Hook->do_action(Array)
#3 E:\[domain]\www\wp-includes\general-template.php(2630): do_action('wp_footer')
#4 E:\[domain]\www\wp-content\themes\Divi\footer.php(58): wp_footer()
#5 E:\[domain]\www\wp-includes\template.php(688): require_once('E:\\[domain]\\w...')
#6 E:\[domain]\www\wp-includes\template.php(647): load_template('E:\\[domain]\\w...', true)
#7 E:\[domain]\www\wp-includes\general-template.php(76): locate_template(Array, true)
#8 E:\[domain]\www\wp-content\themes\Divi\404.php(19): get_footer()
#9 E:\[domain]\www\wp-content\plugins\wordfence\lib\wordfenceClass.php(6433): include('E:\\[domain]\\w...')
# in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php on line 226
[08-Nov-2018 02:10:08 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function et_fb_enabled() in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php:226
Stack trace:
#0 E:\[domain]\www\wp-includes\class-wp-hook.php(286): et_builder_get_modules_js_data('')
#1 E:\[domain]\www\wp-includes\class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 E:\[domain]\www\wp-includes\plugin.php(453): WP_Hook->do_action(Array)
#3 E:\[domain]\www\wp-includes\general-template.php(2630): do_action('wp_footer')
#4 E:\[domain]\www\wp-content\themes\Divi\footer.php(58): wp_footer()
#5 E:\[domain]\www\wp-includes\template.php(688): require_once('E:\\[domain]\\w...')
#6 E:\[domain]\www\wp-includes\template.php(647): load_template('E:\\[domain]\\w...', true)
#7 E:\[domain]\www\wp-includes\general-template.php(76): locate_template(Array, true)
#8 E:\[domain]\www\wp-content\themes\Divi\404.php(19): get_footer()
#9 E:\[domain]\www\wp-content\plugins\wordfence\lib\wordfenceClass.php(6433): include('E:\\[domain]\\w...')
# in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php on line 226
[08-Nov-2018 02:10:10 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function et_fb_enabled() in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php:226
Stack trace:
#0 E:\[domain]\www\wp-includes\class-wp-hook.php(286): et_builder_get_modules_js_data('')
#1 E:\[domain]\www\wp-includes\class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 E:\[domain]\www\wp-includes\plugin.php(453): WP_Hook->do_action(Array)
#3 E:\[domain]\www\wp-includes\general-template.php(2630): do_action('wp_footer')
#4 E:\[domain]\www\wp-content\themes\Divi\footer.php(58): wp_footer()
#5 E:\[domain]\www\wp-includes\template.php(688): require_once('E:\\[domain]\\w...')
#6 E:\[domain]\www\wp-includes\template.php(647): load_template('E:\\[domain]\\w...', true)
#7 E:\[domain]\www\wp-includes\general-template.php(76): locate_template(Array, true)
#8 E:\[domain]\www\wp-content\themes\Divi\404.php(19): get_footer()
#9 E:\[domain]\www\wp-content\plugins\wordfence\lib\wordfenceClass.php(6433): include('E:\\[domain]\\w...')
# in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php on line 226
[08-Nov-2018 02:10:13 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function et_fb_enabled() in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php:226
Stack trace:
#0 E:\[domain]\www\wp-includes\class-wp-hook.php(286): et_builder_get_modules_js_data('')
#1 E:\[domain]\www\wp-includes\class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 E:\[domain]\www\wp-includes\plugin.php(453): WP_Hook->do_action(Array)
#3 E:\[domain]\www\wp-includes\general-template.php(2630): do_action('wp_footer')
#4 E:\[domain]\www\wp-content\themes\Divi\footer.php(58): wp_footer()
#5 E:\[domain]\www\wp-includes\template.php(688): require_once('E:\\[domain]\\w...')
#6 E:\[domain]\www\wp-includes\template.php(647): load_template('E:\\[domain]\\w...', true)
#7 E:\[domain]\www\wp-includes\general-template.php(76): locate_template(Array, true)
#8 E:\[domain]\www\wp-content\themes\Divi\404.php(19): get_footer()
#9 E:\[domain]\www\wp-content\plugins\wordfence\lib\wordfenceClass.php(6433): include('E:\\[domain]\\w...')
# in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php on line 226
[08-Nov-2018 02:10:16 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function et_fb_enabled() in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php:226
Stack trace:
#0 E:\[domain]\www\wp-includes\class-wp-hook.php(286): et_builder_get_modules_js_data('')
#1 E:\[domain]\www\wp-includes\class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 E:\[domain]\www\wp-includes\plugin.php(453): WP_Hook->do_action(Array)
#3 E:\[domain]\www\wp-includes\general-template.php(2630): do_action('wp_footer')
#4 E:\[domain]\www\wp-content\themes\Divi\footer.php(58): wp_footer()
#5 E:\[domain]\www\wp-includes\template.php(688): require_once('E:\\[domain]\\w...')
#6 E:\[domain]\www\wp-includes\template.php(647): load_template('E:\\[domain]\\w...', true)
#7 E:\[domain]\www\wp-includes\general-template.php(76): locate_template(Array, true)
#8 E:\[domain]\www\wp-content\themes\Divi\404.php(19): get_footer()
#9 E:\[domain]\www\wp-content\plugins\wordfence\lib\wordfenceClass.php(6433): include('E:\\[domain]\\w...')
# in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php on line 226
[08-Nov-2018 02:10:19 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function et_fb_enabled() in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php:226
Stack trace:
#0 E:\[domain]\www\wp-includes\class-wp-hook.php(286): et_builder_get_modules_js_data('')
#1 E:\[domain]\www\wp-includes\class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 E:\[domain]\www\wp-includes\plugin.php(453): WP_Hook->do_action(Array)
#3 E:\[domain]\www\wp-includes\general-template.php(2630): do_action('wp_footer')
#4 E:\[domain]\www\wp-content\themes\Divi\footer.php(58): wp_footer()
#5 E:\[domain]\www\wp-includes\template.php(688): require_once('E:\\[domain]\\w...')
#6 E:\[domain]\www\wp-includes\template.php(647): load_template('E:\\[domain]\\w...', true)
#7 E:\[domain]\www\wp-includes\general-template.php(76): locate_template(Array, true)
#8 E:\[domain]\www\wp-content\themes\Divi\404.php(19): get_footer()
#9 E:\[domain]\www\wp-content\plugins\wordfence\lib\wordfenceClass.php(6433): include('E:\\[domain]\\w...')
# in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php on line 226
[08-Nov-2018 02:10:21 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function et_fb_enabled() in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php:226
Stack trace:
#0 E:\[domain]\www\wp-includes\class-wp-hook.php(286): et_builder_get_modules_js_data('')
#1 E:\[domain]\www\wp-includes\class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 E:\[domain]\www\wp-includes\plugin.php(453): WP_Hook->do_action(Array)
#3 E:\[domain]\www\wp-includes\general-template.php(2630): do_action('wp_footer')
#4 E:\[domain]\www\wp-content\themes\Divi\footer.php(58): wp_footer()
#5 E:\[domain]\www\wp-includes\template.php(688): require_once('E:\\[domain]\\w...')
#6 E:\[domain]\www\wp-includes\template.php(647): load_template('E:\\[domain]\\w...', true)
#7 E:\[domain]\www\wp-includes\general-template.php(76): locate_template(Array, true)
#8 E:\[domain]\www\wp-content\themes\Divi\404.php(19): get_footer()
#9 E:\[domain]\www\wp-content\plugins\wordfence\lib\wordfenceClass.php(6433): include('E:\\[domain]\\w...')
# in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php on line 226
[08-Nov-2018 02:10:24 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function et_fb_enabled() in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php:226
Stack trace:
#0 E:\[domain]\www\wp-includes\class-wp-hook.php(286): et_builder_get_modules_js_data('')
#1 E:\[domain]\www\wp-includes\class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 E:\[domain]\www\wp-includes\plugin.php(453): WP_Hook->do_action(Array)
#3 E:\[domain]\www\wp-includes\general-template.php(2630): do_action('wp_footer')
#4 E:\[domain]\www\wp-content\themes\Divi\footer.php(58): wp_footer()
#5 E:\[domain]\www\wp-includes\template.php(688): require_once('E:\\[domain]\\w...')
#6 E:\[domain]\www\wp-includes\template.php(647): load_template('E:\\[domain]\\w...', true)
#7 E:\[domain]\www\wp-includes\general-template.php(76): locate_template(Array, true)
#8 E:\[domain]\www\wp-content\themes\Divi\404.php(19): get_footer()
#9 E:\[domain]\www\wp-content\plugins\wordfence\lib\wordfenceClass.php(6433): include('E:\\[domain]\\w...')
# in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php on line 226
[08-Nov-2018 02:10:27 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function et_fb_enabled() in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php:226
Stack trace:
#0 E:\[domain]\www\wp-includes\class-wp-hook.php(286): et_builder_get_modules_js_data('')
#1 E:\[domain]\www\wp-includes\class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 E:\[domain]\www\wp-includes\plugin.php(453): WP_Hook->do_action(Array)
#3 E:\[domain]\www\wp-includes\general-template.php(2630): do_action('wp_footer')
#4 E:\[domain]\www\wp-content\themes\Divi\footer.php(58): wp_footer()
#5 E:\[domain]\www\wp-includes\template.php(688): require_once('E:\\[domain]\\w...')
#6 E:\[domain]\www\wp-includes\template.php(647): load_template('E:\\[domain]\\w...', true)
#7 E:\[domain]\www\wp-includes\general-template.php(76): locate_template(Array, true)
#8 E:\[domain]\www\wp-content\themes\Divi\404.php(19): get_footer()
#9 E:\[domain]\www\wp-content\plugins\wordfence\lib\wordfenceClass.php(6433): include('E:\\[domain]\\w...')
# in E:\[domain]\www\wp-content\themes\Divi\includes\builder\framework.php on line 226

**Comment by MIBul** [comment]
Anybody looking at this issue?
Just as an update: after disabling the wordfence plugin the errors are gone.

**Comment by lots0logs** [comment]
Hi, we actually do not provide support on GitHub. If you could, please open a support ticket with our team. You can do that by logging into our website and clicking the pink icon at the bottom right corner of the screen. Cheers!

**Comment by albiurs** [comment]
Hi, I actually did already ask for support 2 or 3 moths ago, but the guy answering to my mail unfortunately did not consider it as important enough to pass the issue to the dev team (or he didn't really understand what it is about). As it's obviously a bug which over 2000 sites show in their frontend according to a Google search (see above search - line number: 220 and 226), it's not something we need support for as individual users. I bet there are over 100'000 sites having the same issue but only these 2000+ show it in the front end, as they have wp_debug activated in the wp_config.php. I think there should be rather an effort from the dev team to actually study the issue and provide a fix. For me it's not clear how to get trough to the dev team to report a bug if the support guys do not listen to users or not understand the point of the ticket... Is there a proper place to report Divi-related bugs? Cheers!

**Comment by lots0logs** [comment]
Hi, if this bug were affecting as many sites as you claim then I guarantee you we would have heard about it before now. We can certainly look into it though. The way to do that is to open a support ticket and then the support rep will verify your issue and open an issue on GiutHub so the dev team can take a look :smile:

**Comment by albiurs** [comment]
Hi, Opening a support ticket as you suggest is what I already did, but I didn't get to the ET 1st level support firewall because the issue is not directly reproducible by a specific action. It happens randomly every couple of days and of course I can't take a whole multisite with various customer websites offline for a whole week by deactivating all the plugins as the support guy suggested.

Actually you didn't hear about the issue, because allmost nobody logs php errors over time and from those who do, almost nobody is actually reading and tracking the errors. I only noticed the error because I do both, logging and tracking php messages.

Further, it's not about claiming, as the issue actually exists on tons of sites. The google search from my firs post currently shows 1440 results with this issue. And if you change the line number in the search to 226 (because the affected line changed due to recent Divi updates), there are 510 more pages with the same issue, so around 2000 in total. However, these are only the sites showing the issue in the frontend because they most likely have set "define('WP_DEBUG', true);" in their wp-config.php. But this is by sure not the case for most sites! Most websites like my own sites do not show any php errors in the frontend, so the issue will not get indexed by Google at all. Therefore, the dark figure of affected websites must be by far higher and the factor can only be estimated based on the estimated percentage of sites with active WP_DEBUG ...
